server/views/pages/prisonerList/index.njk

```diff
@@ -24,5 +24,3 @@
   </div>
 
 {% endblock %}
-
-{% block mojFrontendInit %}{% endblock %}
```

Comment:
This was used to stop the MOJ frontend JS library from initialising on the prisoner list page. This prevented the MOJ Sortable Table JS from loading and breaking the custom sort.

A better way to do this is to simply remove the `data-module=="moj-sortable-table"` attribute from the table itself, which I have done.